lisa/nic.py

```diff
@@ -119,6 +119,8 @@ class Nics(InitializableMixin):
         )
     )
 
+    __file_not_exist = re.compile(r".*No such file or directory", re.MULTILINE)
```

Comment:
This class is not a super class of others. So one underscore is enough to mark private variables.

Reply:
The leading ".*" should be able to remove, it doesn't affect in this pattern.

Reply:
update for both comments